### ISSUE TITLE
Thanks & How to replicate this plot

### ISSUE THREAD

**Issue by KaleabTessera** [ericyuxuanye/Multi-Agent-Craftax#2]
Thanks for making this open source! 

Just wanted to know how to generate the results that led to this plot?

![Image](https://github.com/user-attachments/assets/9c657aac-62fe-4e01-98bb-ccb6b1dc8dae)

I know there is the LSTM and MLP training code [here](https://github.com/ericyuxuanye/Multi-Agent-Craftax/tree/d2a76a2172726fb47b4376a83b9271dfdd8393b8/craftax/craftax_classic/train). I know the hyperparams from the paper seem a little different to the code here (e.g. num envs) and I also don't see achievements being logged, did you do something like `env_state.achievements.sum(axis=-1).mean()`?

**Comment by ericyuxuanye** [comment]
Thank you for your interest! Over the course of training, I saved the model parameters every 20 iterations, and after training, I ran each of those parameters again on 50 (If I recall correctly) instances of the environment, taking the mean and standard deviation.

**Comment by KaleabTessera** [comment]
Awesome thanks for info. 

What hyperparams did you use for the RNN and FF runs? I tried using the ones from paper, but couldn't get info on all hyperparams.

**Comment by ericyuxuanye** [comment]
Of course! For the RNN values I used the defaults from metacontroller_jax.py, except for these values

| Parameter    | Value |
| -------- | ------- |
| `num_envs`  |  500   |
| `num_iterations` | 400  |
| `update_epochs`    | 5   |
| `learning_rate` | 2.5e-4 |
| `num_steps` | 500 |
| `anneal_lr` | `False` |
| `num_minibatches` | 10 |
| `static_parameters` | `StaticEnvParams(num_players=4)` |
| `max_grad_norm` | 1.0 |

So something like this:

```python
from craftax.craftax_classic.train.metacontroller_jax import ClassicMetaController
from craftax.craftax_classic.envs.craftax_state import StaticEnvParams

metacontroller = ClassicMetaController(
    static_parameters=StaticEnvParams(num_players=4),
    num_envs=500,
    num_minibatches=10,
    num_steps=500,
    num_iterations=400,
    update_epochs=5,
    anneal_lr=False,
    learning_rate=2.5e-4,
    max_grad_norm=1.0,
)

params, opt_states, log = metacontroller.train()
```

I believe I used the same values for the feed forward network as well.

**Comment by KaleabTessera** [comment]
Okay, great thank you!
